### PR TITLE
Depl-158762 Add login baner for SOC2 compliance

### PR DESCRIPTION
**Pull request by KupchinskiyVasiliy**

### CODE REVIEW

File: setup/initial/setup-utils.sh [hunk 1]
```diff
@@ -147,6 +148,21 @@ EOF
...
 }
 
+# Uses AWS_DOMAIN, SCRIPTS_PATH variables
+function update_motd() {
+    if [ -z "$AWS_DOMAIN" ]; then
+        return 0
+    fi
+
+    if [ "$AWS_DOMAIN" != "ov.internal" ]; then
+        # Ignore 99-warning from match files to copy
```
Comment (asmoiseenko) [inline setup/initial/setup-utils.sh:158]: I think we should install all mot.d files regardless aws_domain value. If other customers will use scripts in their aws accounts they may fork and customize banner and warnings

Comment (KupchinskiyVasiliy) [reply setup/initial/setup-utils.sh:158]: @KeyMasterOfGozer agree?

Comment (KeyMasterOfGozer) [reply setup/initial/setup-utils.sh:158]: Maybe we should keep an easily modified section of the script with OneVizion specific things that a customer could modify for their needs?  The customer may have their own standards for these type of things.

Comment (asmoiseenko) [reply setup/initial/setup-utils.sh:158]: I think, these scripts cannot be used as is by other customers anyway, but can be a good starting point. We provide good practices, like SOC2 compliant warning, which customer can modify among other things by forking our public repository

Comment (asmoiseenko) [reply setup/initial/setup-utils.sh:158]: Also, when starting instances in dev account, I'm not using AWS_DOMAIN, but I still want warning to be adde

Comment (asmoiseenko) [reply setup/initial/setup-utils.sh:158]: @KeyMasterOfGozer, we have to agree on changes and complete review to move forward

Comment (KeyMasterOfGozer) [reply setup/initial/setup-utils.sh:158]: Sure, it's fine.

**Comment by asmoiseenko** [comment]
@KeyMasterOfGozer you and me are reviewers here we shouldn't merge pr. We provide our comments and approve when satisfied with the changes.
@KupchinskiyVasiliy please implement discussed changes in master

**Pull request merged**